### PR TITLE
HOTFIX: username-duplication

### PR DESCRIPTION
**Pull request by co012**
Someone could create user with already taken username by changing some letter to upper case.

### CODE REVIEW

**Review by alejomendoza — APPROVED** [review]

File: src/graphql/create-user-with-email.ts [hunk 1]
```diff
@@ -33,6 +33,10 @@ const createUserWithEmail = {
     if (!args.username) {
       throw `Username can't be an empty string`;
     }
+
+    // TODO: Make a proper fix
+    args.username = args.username.toLowerCase();
+
```
Comment (jmysliv) [inline src/graphql/create-user-with-email.ts:39]:
```suggestion
```

I'd remove it if the username is not required to be lowercase.

Comment (alejomendoza) [reply src/graphql/create-user-with-email.ts:39]: we should require usernames to be lower case

**Comment by jmysliv** [comment]
@alejomendoza if the usernames should be lowercase we need to make a proper adjustment in the frontend to allow only lowercase while typing username in the form. But yes, I think we can merge it.

**Comment by alejomendoza** [comment]
sounds good will add it as a task @jmysliv

**Pull request merged**